templates/taxi/manufacturer_list.html

```diff
@@ -8,6 +9,11 @@ <h1>
     </a>
   </h1>
 
+  <form action="" method="get" class="form-inline">
+    {{ search_form|crispy }}
+    <input type="submit" value="🔍" class="btn btn-secondary">
+  </form>
```

Comment:
Duplicate code in all files. Create additional include template and reuse it in all places